query.go

```diff
@@ -37,6 +37,12 @@ var (
 	// before this delay, we assume the TX was accepted.
 	QueryRejectTimeout = time.Second
 
+	// QueryInvalidTxThreshold is the threshold for the fraction of peers
+	// that need to respond to a TX with a code of pushtx.Invalid to count
+	// it as invalid, even if not all peers respond. This currently
+	// corresponds to 50% of peers that need to reject.
+	QueryInvalidTxThreshold float32 = 0.5
```

Comment:
Perhaps we should raise this to 60%? Lofty reasoning, but then this would require a super majority for us to give up (keeping in mind that we select _all_ our peers manually as they're outbound connections).

Reply:
Hard to pick a value indeed. I was mostly thinking of a scenario with just two peers, then one would be enough. But anyway, bumped it to 60% and also made it configurable through the query options.